app/models/concerns/searchable.rb

```diff
@@ -8,7 +8,8 @@ module Searchable
 
     index_name [Rails.application.engine_name, Rails.env].join('_')
 
-    def self.search(query, filter_countries, filter_states, filter_cities, filter_lang, with_explain)
+    def self.search(query, filter_regions, filter_countries, filter_states, filter_cities, filter_lang, with_explain)
```

Comment:
@sabrina-ulbrich war das nicht schon mal in einem anderen PR? Hat ja jetzt mit den Kateforien nichts zu tun, oder?

Reply:
@zaziemo war mein fehler, hatte das wieder rausgenommen, ist aber notwendig für die regionale suche, deshalb wieder reingenommen :|

Reply:
oki :)